pkg/services/store/auth.go

```diff
@@ -10,6 +10,12 @@ import (
 	"github.com/grafana/grafana/pkg/services/user"
 )
 
+type testUserKey struct{}
+
+func ContextWithUser(ctx context.Context, data *user.SignedInUser) context.Context {
```

Comment:
question, not necessarily to be done in this PR: should we use this method in the gRPC server?